changelog/_go-ver-1150.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:change
-core: Bump Go version to 1.21.1.
+core: Bump Go version to 1.21.9.
```

Comment:
This should be in a new file `changelog/_go-ver-1157.txt` that matches the Vault minor version, not updating an existing file (only on main do we update the go version file )